model/src/plugin_handler.cpp

```diff
@@ -934,7 +951,8 @@ bool PluginHandler::InstallPlugin(const std::string& path,
                                   bool only_metadata) {
   if (!extractTarball(path, filelist, metadata_path, only_metadata)) {
     std::ostringstream os;
-    os << "Cannot unpack plugin tarball at : " << path;
+    wxLogWarning("Cannot unpack plugin tarball at %s, check if tarball contains metadata.xml",
```

Comment:
Why remove the os << message  in favor of the rather antiquated wsLogWarning?

" check if tarball contains metadata.xml" is IMHO not appropriate here -  please stick to known facts which is that we cannot unpack the tarball. Again, this should be wxLogMessage()

The simple fix would be to leave things in place and just add "wxLogMessage(os.str());"